wgsl/index.bs

```diff
@@ -13437,66 +13440,75 @@ Reads a single texel from a texture without sampling or filtering.
   </thead>
   <tr algorithm="textureLoad 1d">
     <td><var ignore>C</var> is [=i32=] or [=u32=]<br>
+        <var ignore>L</var> is [=i32=] or [=u32=]<br>
         <var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]
     <td><xmp highlight=rust>
 fn textureLoad(t: texture_1d<ST>,
                coords: C,
-               level: C) -> vec4<ST></xmp>
+               level: L) -> vec4<ST></xmp>
 
   <tr algorithm="textureLoad 2d">
     <td><var ignore>C</var> is [=i32=] or [=u32=]<br>
         <var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]
     <td><xmp highlight=rust>
 fn textureLoad(t: texture_2d<ST>,
                coords: vec2<C>,
-               level: C) -> vec4<ST></xmp>
+               level: L) -> vec4<ST></xmp>
 
   <tr algorithm="textureLoad 2d array">
     <td><var ignore>C</var> is [=i32=] or [=u32=]<br>
+        <var ignore>A</var> is [=i32=] or [=u32=]<br>
+        <var ignore>L</var> is [=i32=] or [=u32=]<br>
         <var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]
     <td><xmp highlight=rust>
 fn textureLoad(t: texture_2d_array<ST>,
               coords: vec2<C>,
-              array_index: C,
-              level: C) -> vec4<ST></xmp>
+              array_index: A,
+              level: L) -> vec4<ST></xmp>
 
   <tr algorithm="textureLoad 3d">
     <td><var ignore>C</var> is [=i32=] or [=u32=]<br>
+        <var ignore>L</var> is [=i32=] or [=u32=]<br>
         <var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]
     <td><xmp highlight=rust>
 fn textureLoad(t: texture_3d<ST>,
                coords: vec3<C>,
-               level: C) -> vec4<ST></xmp>
+               level: L) -> vec4<ST></xmp>
```

Comment:
Please can we apply these names consistently to all texture builtins?`textureDimensions` and friends are still using `C` for `level`.

Reply:
I just implemented this for tint. Maybe this is helpful:
https://dawn-review.googlesource.com/c/dawn/+/107160/1/src/tint/intrinsics.def